pkg/operator/staticpod/controller/installer/installer_controller.go

```diff
@@ -857,3 +891,23 @@ func mirrorPodNameForNode(staticPodName, nodeName string) string {
 func statusConfigMapNameForRevision(revision int32) string {
 	return fmt.Sprintf("%s-%d", statusConfigMapName, revision)
 }
+
+func backOffDuration(count int) time.Duration {
+	d := time.Second * time.Duration(float64(10)*math.Pow(1.5, float64(count)))
```

Comment:
Any particular reason not to use client-go's backoff?

Reply:
I looked, but wasn't sure how it fits. This is simple enough to live on its own.

Reply:
Sure, no objections on that, I was just curious.